.eslintrc.js

```diff
@@ -398,13 +421,11 @@ module.exports = {
       },
     },
     {
-      files: ['ui/components/multichain/**/*.{js,ts,tsx}'],
+      files: ['ui/components/multichain/**/*.{js}'],
       extends: [
         path.resolve(__dirname, '.eslintrc.base.js'),
         path.resolve(__dirname, '.eslintrc.node.js'),
         path.resolve(__dirname, '.eslintrc.babel.js'),
-        path.resolve(__dirname, '.eslintrc.typescript-compat.js'),
-        '@metamask/eslint-config-typescript',
```

Comment:
removed because there are no typescript files here, and to apply the typescript parser we need to have a tsconfig that applies to the files in this list, which we don't have.